docs/rules/no-effect-decorator-and-creator.md

```diff
@@ -1 +1,39 @@
 # no-effect-decorator-and-creator
+
+> This rule disallows using the `@Effect` decorator and the `createEffect` function simultaneously
+
+## Rule Details
+
+There are two ways we can register an Effect in NgRx. One is using the `@Effect` decorator, the other is with the help of the `createEffect` function. Using both simultaneously will result in the Effect being registered twice, and the side-effect will be performed twice every time the corresponding action is dispatched.
```

Comment:
```suggestion
There are two ways we can register an Effect in NgRx. One is using the `@Effect` decorator (this is currently deprecated), the other is with the help of the `createEffect` function. Using both simultaneously will result in the Effect being registered twice, and the side-effect will be performed twice every time the corresponding action is dispatched.
```

Reply:
Very nice! I thought about adding a hint that using `@Effect` is not good either, but somehow forgot